prepare transaction/Shell Script/holacontrato.sh

```diff
@@ -0,0 +1,13 @@
+holacontrato_lacchain() {
```

Comment:
change function name for something more descriptive like llamar_holacontrato or ejecutar_hola something like that